README.md

```diff
@@ -75,6 +78,34 @@ require 'avrolution/rake/check_compatibility_task'
 Avrolution::Rake::CheckCompatibilityTask.define
 ```
 
+### Avro Register Schemas Rake Task
+
+There is a rake task to register new schemas.
+
+For Rails applications, the `avro:register_schemas` task is automatically
+defined via a Railtie.
+
+This rake task requires a comma-separated list of files for the schemas to register.
+
+```bash
+rake avro:register_schemas schemas=/app/avro/schemas/one.avsc,/app/avro/schema/two.avsc
```

Comment:
We could certainly add one, but I don't see an immediate use case for it, so I'm inclined to leave it out for now.